packages/fxa-settings/src/lib/reliers/relier-factory.ts

```diff
@@ -113,7 +113,10 @@ export class RelierFactory {
 
     // Run final validation. This will ensure that the all fields decorated with an @bind are in the
     // the correct state.
-    relier?.validate();
+
+    // Commenting this out so that pages will stop erroring out when we don't have sufficient query params.
+    // This might be a TODO to restore this once we have all the data we need in the React app.
+    // relier?.validate();
```

Comment:
Might let @dschom know for awareness, hmm

Reply:
I have! I commented it out at his advisement!